otterdog/eclipse-zenoh.jsonnet

```diff
@@ -281,173 +289,183 @@ orgs.newOrg('eclipse-zenoh') {
+          deployment_branch_policy: 'selected',
         },
       ],
       rulesets: [
-        customRuleset("main"),
+        customRuleset('main'),
       ],
     },
     orgs.newRepo('zenoh-kotlin') {
       allow_auto_merge: true,
       allow_merge_commit: true,
       allow_update_branch: false,
       delete_branch_on_merge: false,
-      gh_pages_build_type: "legacy",
-      gh_pages_source_branch: "gh-pages",
-      gh_pages_source_path: "/",
+      dependabot_security_updates_enabled: true,
+      gh_pages_build_type: 'legacy',
+      gh_pages_source_branch: 'gh-pages',
+      gh_pages_source_path: '/',
       has_projects: false,
       web_commit_signoff_required: false,
       workflows+: {
-        default_workflow_permissions: "write",
+        default_workflow_permissions: 'write',
       },
       environments: [
         orgs.newEnvironment('github-pages') {
           branch_policies+: [
-            "gh-pages",
-            "main"
+            'gh-pages',
+            'main',
           ],
-          deployment_branch_policy: "selected",
+          deployment_branch_policy: 'selected',
         },
       ],
       rulesets: [
-        customRuleset("main"),
+        customRuleset('main'),
       ],
     },
     orgs.newRepo('zenoh-pico') {
       allow_auto_merge: true,
       allow_merge_commit: true,
       allow_update_branch: false,
       delete_branch_on_merge: false,
-      description: "Eclipse zenoh for pico devices",
+      dependabot_security_updates_enabled: true,
+      description: 'Eclipse zenoh for pico devices',
       web_commit_signoff_required: false,
       workflows+: {
-        default_workflow_permissions: "write",
+        default_workflow_permissions: 'write',
       },
       webhooks: [
         orgs.newRepoWebhook('https://readthedocs.org/api/v2/webhook/zenoh-pico/263750/') {
-          content_type: "json",
+          content_type: 'json',
           events+: readTheDocsWebhookEvents,
-          secret: "pass:bots/iot.zenoh/readthedocs.org/zenoh-pico-webhook-secret"
+          secret: 'pass:bots/iot.zenoh/readthedocs.org/zenoh-pico-webhook-secret',
         },
       ],
+      rulesets: [
+        customRuleset('main'),
+      ],
     },
     orgs.newRepo('zenoh-plugin-dds') {
       allow_auto_merge: true,
       allow_merge_commit: true,
       allow_update_branch: false,
       delete_branch_on_merge: false,
-      description: "A zenoh plug-in that allows to transparently route DDS data.  This plugin can be used by DDS applications, such as ROS2 robotic applications and others, to leverage the zenoh for geographical routing or for better scaling discovery.",
-      homepage: "",
+      dependabot_security_updates_enabled: true,
+      description: 'A zenoh plug-in that allows to transparently route DDS data.  This plugin can be used by DDS applications, such as ROS2 robotic applications and others, to leverage the zenoh for geographical routing or for better scaling discovery.',
```

Comment:
For robotics application we should point to the ROS2 plugin. Can you please ask @JEnoch to update the text and add a link to the proper repo?

Reply:
How about:
```
A zenoh plug-in that allows to transparently route DDS data.  This plugin can be used by DDS applications to leverage zenoh for geographical routing or for better scaling discovery. For ROS2 robotic applications, use https://github.com/eclipse-zenoh/zenoh-plugin-ros2dds
```